gw_eccentricity/eccDefinition.py

```diff
@@ -1480,10 +1560,29 @@ def get_rational_fit_for_extrema(self, x, y, data_name=None):
                     self.special_interp_kwargs_for_omega_gw_extrema["denom_degree"] -= 1
                 if self.special_interp_kwargs_for_omega_gw_extrema["num_degree"] == 1 \
                     and self.special_interp_kwargs_for_omega_gw_extrema["denom_degree"] == 1:
+                    if (self.extra_kwargs["use_only_these_many_orbits"]
```

Comment:
we may want `self.extra_kwargs["use_only_these_many_orbits"] is not None` here to avoid confusion?